wifi/wpa_supplicant.conf

```diff
@@ -0,0 +1,2 @@
+ctrl_interface=DIR=/data/misc/wifi/wpa_supplicant GROUP=wifi
+driver_param=test_dir=/data/misc/wifi/sockets
```

Comment:
Sorry I forgot to mention the need of [1] to this file. It lets wpa_supplicant to be able to update config.

[1] https://github.com/mozilla-b2g/device_generic_goldfish/commit/3cae7eee1f7f00ba1eb53c733a8779f345055c6e